CMakeLists.txt

```diff
@@ -128,7 +131,7 @@ endif()
 
 
 # If the user did not customize the install prefix,
-# set it to live under build so we don't inadverently pollute /usr/local
+# set it to live under build so we don't inadvertently pollute /usr/local
```

Comment:
misspelling

Reply:
Lol.  That has been like that for 5 1/2 years.  I don't even know how many times I've read through this file and not noticed the misspelling.